bin/git-delete-submodule

```diff
@@ -34,13 +34,16 @@ git add '.gitmodules'
 [  "$(wc -l '.gitmodules' | cut -d' ' -f1)" = '0' ] && git rm -f '.gitmodules'
 
 # 3. Need to confirm and commit the changes for yourself
-if git submodule status >/dev/null 2>&1 \
-	&& ! git submodule status | grep "$NAME"; then
+git_status_text="$(git submodule status >/dev/null 2>&1)"
```

Comment:
Err, now the stdout of `git submodule status` goes to `/dev/null` too.

Reply:
fixed